stable/prometheus-adapter/README.md

```diff
@@ -49,6 +49,7 @@ The following table lists the configurable parameters of the Prometheus Adapter
 | `rbac.create`                   | If true, create & use RBAC resources                                            | `true`                                      |
 | `resources`                     | CPU/Memory resource requests/limits                                             | `{}`                                        |
 | `rules.default`                 | If `true`, enable a set of default rules in the configmap                       | `true`                                      |
+| `rules.resources`               | If `true`, enable resource metric collection in the configmap                   | `false`                                     |
```

Comment:
I think you should add a section to the README that details what enabling this truly means and links to other relevant docs. It's not clear from just this statement that it allows prometheus-adapter to provide all of the functionality of metrics-server by implementing the metrics resource API. And that doing so enable `kubectl top` and HPA will work.